### PR TITLE
[SPARK-16028][SPARKR] spark.lapply can work with active context

### PR DESCRIPTION
**Pull request by felixcheung**
## What changes were proposed in this pull request?

spark.lapply and setLogLevel
## How was this patch tested?

unit test

@shivaram @thunterdb

### CODE REVIEW

**Comment by felixcheung** [comment]
example update in the other PR.

File: R/pkg/R/context.R [hunk 1]
```diff
@@ -252,17 +252,19 @@ setCheckpointDir <- function(sc, dirName) {
...
 #' @rdname spark.lapply
-#' @param sc Spark Context to use
 #' @param list the list of elements
 #' @param func a function that takes one argument.
 #' @return a list of results (the exact type being determined by the function)
 #' @export
 #' @examples
 #'\dontrun{
-#' sc <- sparkR.init()
-#' doubled <- spark.lapply(sc, 1:10, function(x){2 * x})
```
Comment (shivaram) [inline R/pkg/R/context.R]: can we put in `spark.session()` here to show that we need to do some initialization ?

**Comment by shivaram** [comment]
LGTM. Minor comment about docs

**Comment by felixcheung** [comment]
Updated. thx

**Comment by shivaram** [comment]
Thanks - merging this to master and branch-2.0 after jenkins passes

**Comment by sun-rui** [comment]
I think spark.lapply() is a case that demonstrates the need for supporting Dataset in SparkR. Removing the explicit sc parameter is quite helpful to moving to Dataset internally in the future if Dataset is to be supported

**Pull request closed without merging**